packages/aws-cdk-lib/aws-stepfunctions-tasks/lib/emr/emr-create-cluster.ts

```diff
@@ -154,6 +154,14 @@ export interface EmrCreateClusterProps extends sfn.TaskStateBaseProps {
    * @default true
    */
   readonly visibleToAllUsers?: boolean;
+
+  /**
+   * The amount of idle time in seconds after which the cluster automatically terminates.
+   * You can specify a minimum of 60 seconds and a maximum of 604800 seconds (seven days).
```

Comment:
A line break would be good.

```suggestion
   * 
   * You can specify a minimum of 60 seconds and a maximum of 604800 seconds (seven days).
```